gtsam/navigation/CombinedImuFactor.h

```diff
@@ -24,14 +24,19 @@
 
 /* GTSAM includes */
 #include <gtsam/navigation/PreintegrationCombinedParams.h>
+#include <gtsam/navigation/LieGroupPreintegration.h>
 
 namespace gtsam {
 
 #ifdef GTSAM_TANGENT_PREINTEGRATION
 typedef TangentPreintegration PreintegrationType;
 #else
+#ifdef GTSAM_LIEGROUP_PREINTEGRATION
```

Comment:
We might want to consider deprecating using compile-time flags in favor of a run-time mechanism.